charts/incubator/tt-rss/values.yaml

```diff
@@ -0,0 +1,179 @@
+  pullPolicy: IfNotPresent
+  tag: 283ad4eb@sha256:f90470ec756dbe8ed9e7f842f19f9c1d7ed4a20aca0a1886e9694b3e7fa82bf7
+securityContext:
+  container:
+    readOnlyRootFilesystem: false
+    runAsNonRoot: false
+    runAsUser: 0
+    runAsGroup: 0
+persistence:
+  html:
+      enabled: true
+      targetSelector:
+        main:
+          main:
+            mountPath: /var/www/html
+        nginx:
+          nginx:
+            mountPath: /var/www/html
+            readOnly: true
+        updater:
+          updater:
+            mountPath: /var/www/html
+  config:
+    enabled: true
+    targetSelector:
+        main:
+          main:
+            mountPath: /opt/tt-rss/config.d
+        updater:
+          updater:
+            mountPath: /opt/tt-rss/config.d
+  # plugins:
+  #   enabled: true
+  #   mountPath: /var/www/html/tt-rss/plugins.local
+  # themes:
+  #   enabled: true
+  #   mountPath: /var/www/html/tt-rss/themes.local
+cnpg:
+  main:
+    enabled: true
+    user: tt-rss
+    database: tt-rss
+portal:
+  open:
+    enabled: true
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          probes:
+            liveness:
+              type: tcp
+              port: "{{ .Values.service.app.ports.app.port }}"
+            readiness:
+              type: tcp
+              port: "{{ .Values.service.app.ports.app.port }}"
+            startup:
+              type: tcp
+              port: "{{ .Values.service.app.ports.app.port }}"
+          env:
+            TTRSS_SELF_URL_PATH: ""
+            # tt-rss tests with "-z" so an empty string should still allow the randomly generated password to be made.
+            ADMIN_USER_PASS: ""
+            TTRSS_DB_NAME: "{{ .Values.cnpg.main.database }}"
+            TTRSS_DB_USER: "{{ .Values.cnpg.main.user }}"
+            TTRSS_DB_PORT: "5432"
+            TTRSS_DB_PASS:
+              secretKeyRef:
+                name: cnpg-main-user
+                key: password
+            TTRSS_DB_HOST:
+              secretKeyRef:
+                name: cnpg-main-urls
+                key: host
+  nginx:
+    enabled: true
+    type: Deployment
+    podSpec:
+      containers:
+        nginx:
+          enabled: true
+          primary: true
+          imageSelector: nginxImage
+          probes:
+            readiness:
+              enabled: true
+              type: tcp
+              port: "{{ .Values.service.main.ports.main.targetPort }}"
+              # path: "{{ .Values.workload.nginx.podSpec.containers.nginx.env.APP_BASE }}/public.php?op=healthcheck"
+              # port: "{{ .Values.service.main.ports.main.targetPort }}"
+            liveness:
+              enabled: true
+              type: tcp
+              port: "{{ .Values.service.main.ports.main.targetPort }}"
+              # path: "{{ .Values.workload.nginx.podSpec.containers.nginx.env.APP_BASE }}/public.php?op=healthcheck"
+              # port: "{{ .Values.service.main.ports.main.targetPort }}"
```

Comment:
you dont need the full url.
Only the `/pub...` part

Reply:
Thank you. Updated in https://github.com/truecharts/charts/pull/17293/commits/e97e28e4a7571205a964e879592c08932e77915a.